docs/upgrade/upgrade-to-enterprise/upgrade-guide-from-5.1-to-2022.2/upgrade-guide-from-5.1-to-2022.1-generic.rst

```diff
@@ -0,0 +1,373 @@
+.. |SCYLLA_NAME| replace:: ScyllaDB
+
+.. |SRC_VERSION| replace:: 5.1
+.. |NEW_VERSION| replace:: 2022.2
```

Comment:
The name of this file is **2022.1** but the content is **2022.2**. I think there's a mistake in the file name given the title of the PR?

Reply:
Its a mistake

Reply:
@nyh on a second look, a second commit to the PR already fixed this issue.